test/e2e.sh

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env sh
```

Comment:
I love this ! 👍

Reply:
I looked around for tools and then thought, "eeeeeh, sod it, shell script will do" :) 

Found a good blogpost about this though: https://spin.atomicobject.com/2016/01/11/command-line-interface-testing-tools/